src/main/webapp/data/developers.json

```diff
@@ -1601,21 +1613,41 @@
       "username": "darrenwee",
       "startPeriod": "Jan 2018"
     },
+    {
+      "name": "Jacob Li",
```

Comment:
You should remove entries about them under contributors.

Reply:
@wkurniawan07 Changes made :)